integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/MockSdxUpgradeTests.java

```diff
@@ -79,6 +79,10 @@ public void testSdxUpgradeWhenNoAttachedDisksButEmbeddedDBShouldHaveBeenOnSelfCo
         String cluster = "cmcluster";
         String imageSettings = "imageSettingsUpgrade";
         String networkKey = "someOtherNetwork";
+        testContext
```

Comment:
I know you added this last time but do you mind explaining why these are needed? We've never needed something like this for mock tests before right? I don't see why your changes would require this for the upgrade tests.